### ISSUE TITLE
more specific informations (time to debug)

### ISSUE THREAD

**Issue by Flashpok** [NEDb-tk/deemix-foobar2000#13]
Hi,
I spend many time to search correct informations (between foobar.txt, strip.txt & link)
Is it possible (One week or two) to write in link.txt for all lines "format(unaccented_string)" AND ".format(title_ascii, artist_ascii, album_ascii))" ?

If the track link is good. For example : link:  xxx /// unaccented_string xxx : /// string.split : xxx

It will be more useful to compare informations.

Tell me if you prefer I make this in my main.py file !

**Comment by NEDb-tk** [comment]
Yeah sure! if you have noticed, throughout the script, there are a ton of comments (`#print(x)`). I will have `main.py` spit out 2 files:
`link.txt` and `log.txt`. 
`log.txt` will contain all debugging information.

**Comment by Flashpok** [comment]
Good idea ! Thank you !
